qa/automation/nikeev/Any Career Portal Page/Cookies.feature

```diff
@@ -0,0 +1,94 @@
+Feature: Cookies settings on any career portal page
+
+Background: 'Necessary cookies' is a required option in Cookies Settings box
```

Comment:
A Background allows you to add some context to the scenarios that follow it. It can contain one or more Given steps, which are run before each scenario

Reply:
Okay, I understand. Ohh, I had to leave the first committed version without Background)